CHANGELOG.md

```diff
@@ -12,6 +12,20 @@ Subheadings to categorize changes are `added, changed, deprecated, removed, fixe
 
 ### Added
 
+...
+
+### Changed
+
+...
+
+### Fixed
+
+...
+
+## 0.1.1 (2024-05-28)
```

Comment:
We might be able to do the release before the 28th, so before merging we should make sure we're ready to publish.

Reply:
Apparently, my vision is blurry at night and I misread `26` for `28`